application/shared-kernel/ApiCore/SinglePageApp/SinglePageAppConfiguration.cs

```diff
@@ -64,7 +64,7 @@ public string GetHtmlTemplate()
             return _htmlTemplate;
         }
 
-        AwaitSinglePageAppGeneration();
+        AwaitSinglePageAppGeneration(60, TimeSpan.FromMilliseconds(500));

```

Comment:
No reason to create parameters, as this is hardcoded anyways. Let's make it simple.